src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+
+import Icon from "../Icon/Icon";
+import { Item, Wrapper, AbsoluteWrapper, Current } from "./Dropdown.styles";
+
+export type SelectOption = {
+  label: string;
+  value: string;
+};
+
+export type DropdownProps = {
+  value?: SelectOption;
+  options: SelectOption[];
+  onChange: (option: SelectOption) => void;
+  width?: number;
+};
```

Comment:
```suggestion
  className?: string;
};
```

Styled-components needs `className` to let a parent style it's children.

https://styled-components.com/docs/basics#styling-any-component